trussworks/react-uswds
# feat: Added multiselect story

# Summary

- Adds story for multiple select
- Adds extra long content in an option for visual testing purposes
- Adds storybook arg to effectively unset the radio buttons for validation styling

## Related Issues or PRs

Addresses https://github.com/uswds/uswds/pull/5268, in that this component was already working fine, but now you can _see_ that it works fine.


## How To Test

View "Multiselect" story for "Select" component, see ellipsis is added already via imported css, and set/unset the validation style in the storybook args

### Screenshots (optional)
<img width="739" alt="image" src="https://github.com/trussworks/react-uswds/assets/764090/61fc2d3a-51ff-4303-a143-b388a342d223">
